style.css

```diff
@@ -311,6 +319,27 @@ details li[class]:after {
 
 .very-buggy:after { content: ':o'; }
 
+.progress-bar {
+    height: 100%;
```

Comment:
You should use tabs for indentation here.

Reply:
Fixed